draft-ietf-rats-architecture.md

```diff
@@ -679,7 +679,7 @@ In the background-check model, this Evidence may also be revealed to Relying Par
 
 ## Evidence
 
-Evidence is a set of claims about the target device that reveals operational status, health, configuration or construction that have security relevance. Evidence is evaluated by a Verifier to establish its relevance, correctness and timeliness. Claims need to be collected in a manner that is reliable, that way if the target somehow can lie about them, the attesting environement is able to detect or circumvent it (lying). Evidence needs to be securely associated with the target device so that the Verifier cannot be tricked into accepting claims originating from a different target (that may be more trustworthy than the intended target). Evidence also must be protected from man-in-the-middle attackers who may observe, change or misdirect Evidence as it travels from Attester to Verifier. The timeliness of Evidence can be captured using claims that pinpoint the time or interval where changes in operational status, health and so forth occur.
+Evidence is a set of claims about the target device that reveal operational status, health, configuration or construction that have security relevance. Evidence is evaluated by a Verifier to establish its relevance, correctness, and timeliness. Claims need to be collected in a manner that is reliable. That way if the target device somehow can lie about them, the attesting environment is able to detect or circumvent it (lying). Evidence needs to be securely associated with the target device so that the Verifier cannot be tricked into accepting claims originating from a different device (that may be more trustworthy). Evidence also must be protected from man-in-the-middle attackers who may observe, change or misdirect Evidence as it travels from Attester to Verifier. The timeliness of Evidence can be captured using claims that pinpoint the time or interval where changes in operational status, health and so forth occur.
```

Comment:
This still shows up as one long line (it's all on line 682).
Please split into multiple source lines.